### DISCUSSION TITLE
Sponsor a LOVE2D Port

### DISCUSSION THREAD

**Discussion by splashsky** [Annoraaq/grid-engine#310] [Ideas]
Hey Anno! Didn't know where else to contact you for something like this.

I'm interested in potentially sponsoring a port of Grid Engine to the [LOVE2D framework](https://love2d.org/). This is a game/graphics rendering framework entirely in Lua, which is a dynamically typed scripting language founded on the concept of tables.

There exists already a seemingly efficient and robust library for the rendering of Tiled maps, [Simple Tiled Implementation](https://github.com/karai17/Simple-Tiled-Implementation). This is great for the rendering of maps, but has very little capacity or utility for actually making the maps interactive, in the way Grid Engine does.

As such, I'd like to know if there's a possibility to sponsor a port to L:OVE2D, and what you would require in resources to accomplish it, or if you're even interested in such a project.

Thanks!

**Comment by Annoraaq** [comment]
Hey, I have heard about Lua but never used it. I personally don't have enough experience in Lua and not enough capacity to do it myself. However, since this is an open source project, everyone interested is free to port it. So if you can find anyone, go for it.
